# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/nsITlsHandshakeListener.idl
# Commit: f69657a7e597
# Full Hash: f69657a7e597e0cbf62f8dbcbebd8e7e230a9975
# Author: Dragana Damjanovic <dd.mozilla@gmail.com>
# Date: 2021-09-22 15:53:06
# Regressor Bug: 1382886
# File Overlap Count: 1
# Description:
#   Bug 1382886 - Add a callback when the handshake is done. r=necko-reviewers,kershaw,keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123601
# ==============================================================================

diff -r e9635ec4198f -r f69657a7e597 netwerk/protocol/http/nsITlsHandshakeListener.idl
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/netwerk/protocol/http/nsITlsHandshakeListener.idl	Wed Sep 22 09:19:30 2021 +0000
@@ -0,0 +1,12 @@
+/* -*- Mode: C++; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*-
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "nsISupports.idl"
+
+[builtinclass, uuid(b4bbe824-ec4c-48be-9a40-6a7339347f40)]
+interface nsITlsHandshakeCallbackListener : nsISupports {
+    [noscript] void handshakeDone();
+};